python/pyspark/testing/utils.py

```diff
@@ -80,7 +83,7 @@ def eventually(condition, timeout=30.0, catch_assertions=False):
     ----------
     condition : function
         Function that checks for termination conditions. condition() can return:
-            - True: Conditions met. Return without error.
+            - True or None: Conditions met. Return without error.
```

Comment:
@HyukjinKwon I made a change here, so that we don't need `return True` for most cases in the future.